Examples/Tests/ElectrostaticDirichletBC/inputs_2d

```diff
@@ -7,6 +7,7 @@ warpx.use_filter = 0
 amr.n_cell = 64 8
 amr.max_grid_size = 32
 amr.max_level = 0
+warpx.use_filter = 0
```

Comment:
This line (and similar ones below) were added in order to avoid modifying the benchmarks - since filtering is now taken into account in the charge deposition.